### PR TITLE
[imagl] correcting minimum visual studio version

### PR DESCRIPTION
**Pull request by Woazim**
This is a simple correction suggested by @prince-chrismc on pull request #4225 (see its [comment](https://github.com/conan-io/conan-center-index/pull/4225#discussion_r560521897))

### CODE REVIEW

**Comment by prince-chrismc** [comment]
🎉 It's actually running on windows =D

**Comment by prince-chrismc** [comment]
> All green in build 4 (`e12e215f5a0d27f061979859cd1f7ae62d150f3f`)! 😊
> 
> * `imagl/0.1.0`: Generated 12 packages (+ 1 invalid config from `build()`). [All logs here](https://c3i.jfrog.io/c3i/misc/logs/pr/4225/4/imagl/0.1.0/)

https://github.com/conan-io/conan-center-index/pull/4225#issuecomment-761227181

**Review by prince-chrismc — APPROVED** [review]

**Review by intelligide — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

File: recipes/imagl/all/conanfile.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def _build_subfolder(self):
     def _compilers_minimum_version(self):
         return {
             "gcc": "9",
-            "Visual Studio": "16.5",
+            "Visual Studio": "16",
```
Comment (SpaceIm) [inline recipes/imagl/all/conanfile.py:35]: you can avoid to lose this information by using this pattern: https://github.com/conan-io/conan-center-index/blob/b5716c14ae9d40509374398a9cccf6e468a0f73f/recipes/sol2/3.x.x/conanfile.py#L33

Comment (Woazim) [reply recipes/imagl/all/conanfile.py:35]: It *should* be OK to remove the ".5". The source code doesn't use new C++ features that are not in first Visual Studio 2019 release. Some newer features are used only if they are present in the compiler. I use some standard c++ definitions to test it.

Nevertheless, thank you for your help. I keep this tip in my mind.

(The ".5" comes from the fact that I was using Visual Studio 16.5 when I started to code imaGL)

Comment (SpaceIm) [reply recipes/imagl/all/conanfile.py:35]: ok, then LGTM ;)

**Review by SpaceIm — APPROVED** [review]

**Pull request merged**